### PR TITLE
Symfony 5 allowed

### PR DESCRIPTION
**Pull request by parijke**
## Subject

I am targeting this branch, because it needs to be compatible with symfony 5.

## Changelog

```markdown

### Added
- Compatibility with Symfony 5
```

### CODE REVIEW

**Comment by VincentLanglet** [comment]
@parijke You need to update the template
Fill the subject and the changelog

**Comment by parijke** [comment]
> @parijke You need to update the template
> Fill the subject and the changelog

I think I did now...

File: composer.json [hunk 1]
```diff
@@ -27,13 +27,13 @@
...
-        "symfony/phpunit-bridge": "^4.3.8",
-        "symfony/property-access": "^3.4 || ^4.0",
-        "symfony/routing": "^3.4 || ^4.0"
+        "symfony/config": "^3.4 || ^4.0 || ^5.0",
+        "symfony/dependency-injection": "^3.4 || ^4.0 || ^5.0",
+        "symfony/http-foundation": "^3.4 || ^4.0 || ^5.0",
+        "symfony/http-kernel": "^3.4 || ^4.0 || ^5.0",
+        "symfony/phpunit-bridge": "^4.3.8 || ^5.0",
+        "symfony/property-access": "^3.4 || ^4.0 || ^5.0",
+        "symfony/routing": "^3.4 || ^4.0 || ^5.0"
```
Comment (phansys) [inline composer.json]: Could we leverage this PR to use `4.3` as the minimum allowed version from `4.x` branch?

```suggestion
        "symfony/config": "^3.4 || ^4.3 || ^5.0",
        "symfony/dependency-injection": "^3.4 || ^4.3 || ^5.0",
        "symfony/http-foundation": "^3.4 || ^4.3 || ^5.0",
        "symfony/http-kernel": "^3.4 || ^4.3 || ^5.0",
        "symfony/phpunit-bridge": "^4.3.8 || ^5.0",
        "symfony/property-access": "^3.4 || ^4.3 || ^5.0",
        "symfony/routing": "^3.4 || ^4.3 || ^5.0"
```

**Comment by greg0ire** [comment]
Thanks @parijke !

**Pull request merged**